db/migrate/20131002004641_create_automatic_exploitation_matches.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAutomaticExploitationMatches < ActiveRecord::Migration
+  def change
+    create_table :automatic_exploitation_matches do |t|
+      t.integer :ref_id
+      t.string :state
+      t.integer :nexpose_data_vulnerability_definition_id
```

Comment:
I agree it's better to have the generator make the association for you, but I see no reason to go back and change this now. It shouldn't make a difference as far as I know. Unlike indices, which will certainly make a difference. :smiley_cat: